src/sentry/snuba/sessions.py

```diff
@@ -480,6 +480,8 @@ def get_release_health_data_overview(
                 }
 
     # Fill in release adoption
+    from sentry.releasehealth import get_release_adoption
+
```

Comment:
Why import in the middle of the function ? 
If it is just to avoid a name clash why not import it into another name.

Reply:
I thought i'd have a circular dependency but actually it seems lazyservicewrapper takes care of that.